postgis/sqldefines.h.in

```diff
@@ -14,13 +14,8 @@
 #define POSTGIS_LIBXML2_VERSION @POSTGIS_LIBXML2_VERSION@
 #define POSTGIS_SFCGAL_VERSION @POSTGIS_SFCGAL_VERSION@
 
-#if POSTGIS_PGSQL_VERSION >= 96
 #define _PARALLEL PARALLEL SAFE
```

Comment:
can we unroll this define please?

Reply:
I was wondering whether to do it or not. Do you think it's worth changing all those lines?

Reply:
I don't know from "worth it"... I'm going to go with "yes". We used to have STRICT and friends in defines, and we took them out, and didn't suffer, so the one time big patch isn't a huge problem, in my mind.

Reply:
Yes. Less copy paste during one-liner debug.

Reply:
I'll do it then :wink:

Reply:
`postgis_brin.sql.in` doesn't have PARALLEL tags, so I'll try to add them in a different commit.